internal/activities/remove_paths_test.go

```diff
@@ -79,10 +79,10 @@ func TestRemovePaths(t *testing.T) {
 
 				return
 			}
-
 			assert.NilError(t, err)
+
 			var res activities.RemovePathsResult
-			future.Get(&res)
+			_ = future.Get(&res)
```

Comment:
This won't be needed after you merge the other PR.

Reply:
@jraddaoui I kept the RemovePaths activity because I think we'll use it in the future.  I am considering combining this functionality into the [removepaths package](https://github.com/artefactual-sdps/temporal-activities/blob/dev/issue-2-remove-files-activity/removefiles/removefiles.go). I'm thinking about making a `RemoveByPath()` method and renaming `Remove()` -> `RemoveByName()`.